jobs/build/tarball-sources/Jenkinsfile

```diff
@@ -70,109 +64,56 @@ node {
-                    ${advisory} > ${filename}
-                """
-            }
+            def output = ""
 
-            stage("rsync") {
-                cmd = "rsync -avz --no-perms --no-owner --no-group ${outdir} ${rcm_guest}"
-                res = commonlib.shell(
-                    script: cmd,
-                    returnAll: true
-                )
-                if(res.returnStatus != 0 || res.stderr != "") {
-                    error "rsync executing failed..."
+            stage("run tarball-sources") {
+                sh "rm -rf ./artcd_working"
+                sh "mkdir -p ./artcd_working"
+                def cmd = [
+                    "artcd",
+                    "-v",
+                    "--working-dir=./artcd_working",
+                    "--config", "./config/artcd.toml",
+                ]
+                if (params.DRY_RUN) {
+                    cmd << "--dry-run"
                 }
-            }
-
-            stage("file ticket to RCM") {
-                // use awk to print out the lines that contain ${advisory}
-                // sth like this:
-                // RHEL-7-OSE-4.2/${advisory}/release/logging-fluentd-container-v4.2.26-202003230335.tar.gz 
-                cmd = """awk '/${advisory}/ { save=\$0 }END{ print save }' ${filename}"""
-                verify = commonlib.shell(
-                        script: cmd,
-                        returnStdout: true
-                )
-
-                cmd = """echo '${verify}' |awk -F '-v' '{print \$2}' |awk -F '-' '{print \$1}' """
-                version = commonlib.shell(
-                        script: cmd,
-                        returnStdout: true
-                )
-
-                currentBuild.description += " ocp ${version}"
-                description = """Hi RCM,
-                    The OpenShift ART team needs to provide sources for `${components}` in https://errata.devel.redhat.com/advisory/${advisory}
-
-                    The following sources are uploaded to ${rcm_guest}
-
-                    ${verify}
-
-                    Attaching source tarballs to be published on ftp.redhat.com as in https://projects.engineering.redhat.com/browse/RCMTEMPL-6549
-                    """
-                withCredentials([usernamePassword(
-                    credentialsId: 'jboss_jira_login',
-                    usernameVariable: 'JIRA_USERNAME',
-                    passwordVariable: 'JIRA_PASSWORD',
-                )]) {
-                    withEnv(["ds=${description}"]){
-                        cmd = "artjira create -summary=\"OCP Tarball sources\" -description=\"${ds}\""
-                        jira = commonlib.shell(
-                            script: cmd,
-                            returnStdout: true
-                        )
+                cmd += [
+                    "tarball-sources",
+                    "--group", "openshift-${params.VERSION}",
+                    "--assembly", params.ASSEMBLY,
```

Comment:
Same question for here, what happens to 3.11

Reply:
3.11 job gets the default value `stream`.